packages/std/src/math/uint128.rs

```diff
@@ -208,6 +208,10 @@ impl Uint128 {
     pub fn saturating_pow(self, other: u32) -> Self {
         Self(self.0.saturating_pow(other))
     }
+
+    pub const fn abs_diff(self, other: Self) -> Self {
+        Self(self.0.abs_diff(other.0))
```

Comment:
You can do this manually to get rid of the Rust 1.60.0 requirement:

```suggestion
        Self(if self.0 < other.0 {
            other.0 - self.0
        } else {
            self.0 - other.0
        })
```